Tasks/a0_my_stack.py

```diff
@@ -19,7 +21,10 @@ def pop():
 
 	:return: popped element
 	"""
-	return None
+	if my_stack != []:
```

Comment:
Лучше проверять другим образом
```suggestion
	if my_stack:
```